File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c/Herzogs.c
#include <stdio.h>

int main(int argc, char *argv[]) {
    for (int i = 1; i <= 100; i++) {
        if (i%3 == 0 && i%5 == 0)
            fprintf(stdout, "fizzbuzz\n");
        else if(i%3 == 0)
            fprintf(stdout, "fizz\n");
        else if(i%5 == 0)
            fprintf(stdout, "buzz\n");
        else
            fprintf(stdout, "%d\n", i);
    }
    return 0;
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/rust/Herzogs.rs

fn main() {
    for num in 1..=100{
        if num%3==0 && num %5==0{
         println!("{} - fizzbuzz",num);
        }else if num%3 == 0{
         println!("{} - fizz",num);
        }else if num%5==0{
         println!("{} - buzz",num);
        }else{
         println!("{}",num);
        }
    }
}
